packages/lb-annotation/src/core/toolOperation/polygonOperation.ts

```diff
@@ -1157,6 +1180,92 @@ class PolygonOperation extends BasicToolOperation {
     return false;
   }
 
+  public moveSelectedPolygons(offset: ICoordinate) {
+    const newPolygonList = _.cloneDeep(this.polygonList);
+    const selectedPolygons = _.cloneDeep(this.dragInfo?.selectedPolygons);
+
+    if (selectedPolygons && selectedPolygons?.length > 0) {
+      newPolygonList.forEach((v) => {
+        const selectedPolygon = selectedPolygons?.find((s) => v.id === s.id);
+
+        if (selectedPolygon) {
+          v.pointList = v.pointList.map((p, index) => ({
+            ...p,
+            x: selectedPolygon?.pointList[index].x + offset.x,
+            y: selectedPolygon?.pointList[index].y + offset.y,
+          }));
+        }
+
+        return v;
+      });
+
+      this.setPolygonList(newPolygonList);
+      this.render();
+    }
+  }
+
+  /**
+   * Update polygon position while enableDrag is true
+   * @param e {MouseEvent}
+   */
+  public onDragMove(e: MouseEvent) {
+    const coordinate = this.getCoordinateUnderZoom(e);
+
+    const { dragTarget, dragPrevCoord, changePointIndex, initPointList, dragStartCoord } = this.dragInfo!;
+
+    this.dragStatus = EDragStatus.Move;
+
+    if (dragTarget === EDragTarget.Plane) {
+      const offset = {
+        x: (coordinate.x - dragStartCoord.x) / this.zoom,
+        y: (coordinate.y - dragStartCoord.y) / this.zoom,
+      };
+      this.moveSelectedPolygons(offset);
+      return;
+    }
+
+    const { selectedPolygon } = this;
+
+    if (!selectedPolygon) {
+      return;
+    }
+
+    /**
```

Comment:
The following code is `getDragOffset` that it is just return the  `offset`. You need to read `PolygonOperation.dragPolygon` to rewrite it.

Reply:
What's the meaning?